SLA.md

```diff
@@ -11,12 +11,11 @@ The author introduces this issue explaining that most people’s experience with
 ### The Age Factor
 There is a view that “an inherent capacity for language” has got its strongest period before puberty and the acquisition of a new L2 is significantly harder after this time. On the other hand, an opposing view illustrates teenagers as more effective L2 learners in comparison to their younger colleagues (e.g. seven-year-olds). Thus, Yule suggests that proper learning of L2 requires a set of factors such as proper age (from ten to sixteen).
 ### Affective Factors
-There is a possibility that the aforementioned optimum age for learning the second language will be blocked by another issue. As teenagers are more self-conscious, there might be a possibility for them to get embarrassed and, eventually, to get reluctant. Yule describes such negative facets as affective factors creating a barrier between the learner and the second language. One’s mentality is crucial in effective learning and cannot be unmentioned. Affective factors are less serious when it comes to younger learners and this might be why many researchers claim that age matters so much in terms of L2 learning. 
-
+There is a possibility that the aforementioned optimum age for learning the second language will be blocked by another issue. As teenagers are more self-conscious, there might be a possibility for them to get embarrassed and, eventually, to become reluctant. Yule describes such negative facets as affective factors creating a barrier between the learner and the second language. One’s mentality is crucial in effective learning and cannot be unmentioned. Affective factors are less serious when it comes to younger learners and this might be why many researchers claim that age matters so much in terms of L2 learning.
 ## Focus on method
-
+With the problem of the barriers emerged the need to create approaches helpful in cherishing L2 learning. In the past, exercise materials developed. Nowadays, a number of different methods and views on improving the L2 learning experience.
```

Comment:
Po barriers przecinek chyba lepiej dać, jak zdanie zaczyna się od 'with' :) I nie jestem pewna czy Ci czegoś nie ucięło w zdaniu zaczynającym się od 'nowadays', bo na końcu brakuje jakiegoś czasownika, może właśnie 'Nowadays, a number of different methods and views on improving the L2 learning experience is (constantly) appearing/arising/emerging'?